test/wasm2js/comments.2asm.js

```diff
@@ -28,33 +28,3 @@ function asmFunc(global, env, buffer) {
 
 var memasmFunc = new ArrayBuffer(65536);
 var retasmFunc = asmFunc({Math,Int8Array,Uint8Array,Int16Array,Uint16Array,Int32Array,Uint32Array,Float32Array,Float64Array,NaN,Infinity}, {abort:function() { throw new Error('abort'); }},memasmFunc);
-
-function asmFunc(global, env, buffer) {
```

Comment:
What is the cause of this unrelated-looking deletion?

Reply:
This is that `comments.wast` test - we test it as a single big module, instead of pieces. So we do get coverage of parsing the non-texty bytes, but we don't see separate (empty - just comments) modules, and don't emit two modules in the output here.